### ISSUE TITLE
Issue with path causing errors at times.

### ISSUE THREAD

**Issue by KGM6911** [FezVrasta/ark-server-tools#617]
I'm getting this error when doing a backup.  I would think it's the // in the path but I can't find where it's adding the extra / in.  Any clue how I can correct this?

```
 1822438033.arktribe                                             [   OK   ]
   1879736999.arktribe                                             [   OK   ]
 Copying ARK tribute tribe files
   *.arktributetribe cp: cannot stat '/home/steam/crossark-center-prim-pve/ShooterGame/Saved//home/steam/crossark-center-prim-pve/ShooterGame/Saved/SavedArk-crossark-center-prim-pve/*.arktributetribe': No such file or directory
cp: cannot stat '/home/steam/crossark-center-prim-pve/ShooterGame/Saved//home/steam/crossark-center-prim-pve/ShooterGame/Saved/SavedArk-crossark-center-prim-pve/*.arktributetribe': No such file or directory
cp: cannot stat '/home/steam/crossark-center-prim-pve/ShooterGame/Saved//home/steam/crossark-center-prim-pve/ShooterGame/Saved/SavedArk-crossark-center-prim-pve/*.arktributetribe.tmptributetribe': No such file or directory
                                                                   [ FAILED ]
 Copying GameUserSettings.ini                                      [   OK   ]
 Copying Game.ini                                                  [   OK   ]
 Compressing Backup                                                [   OK   ]
 Created Backup:  2016-09-26_02.04.14.tar.bz2
```

**Comment by klightspeed** [comment]
This should be fixed in v1.6.02

**Comment by KGM6911** [comment]
Thank you!
